### ISSUE TITLE
Can't click "Clear all" button on brave://downloads/

### ISSUE THREAD

**Issue by JaneX8** [brave/brave-browser#42924]
### Description

I cannot click the Clear all button for quite some time now. It also is behind a scroll bar. I can only tab through to the button and the hit enter. But using the mouse it does not work.

![Image](https://github.com/user-attachments/assets/30081d03-ec1a-468a-a2ea-fa2da0e899d5)

Windows 11.

![Image](https://github.com/user-attachments/assets/db4e0cd2-4ba9-4628-b441-96fc2a45cb3a)

Resolution 1920x1080. 100% scale. Landscape. Both when using a single or multiple (three) displays. Same resolution. Install location: "C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe".

### Steps to reproduce

Have the same OS, Brave version and resolutions

### Actual result

Unclickable "Clear all" button on "brave://downloads/"

### Expected result

Clickable "Clear all" button.

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

1.73.101 Chromium: 131.0.6778.139 (Official Build) (64-bit) 
45dae32a9fd5d259796f6c975ec055564fac7139
Windows 11 Version 23H2 (Build 22631.4602)

### Channel information

- [x] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

Possibly installed using `scoop.sh` or WinGet Package Manager.

**Comment by LegalizeNukes** [comment]
Same issue occurs on macOS.

**Comment by dushmanta05** [comment]
Have the same alignment issue on Linux (Ubuntu 24.04) although the "Clear All" functionality is working.

![Image](https://github.com/user-attachments/assets/7800cf5d-5667-46f4-8f6b-cdad21361de0)

**Comment by emerick** [comment]
I wasn't able to repro on Windows/Mac at various resolutions (I'm always able to interact with the "Clear" button, though it does overlap the toolbar). @brave/qa-team if we have any clear repro steps on this one, please let me know!

**Comment by LaurenWags** [comment]
@emerick unfortunately I haven't been able to reproduce the inability to click the "Clear" button on brave://downloads, I do see the overlap issue though. Hopefully some others from @brave/qa-team can try - perhaps @MadhaviSeelam or @GeetaSarvadnya can look on their Win machines.

```
Brave | 1.73.104 Chromium: 131.0.6778.204 (Official Build) (x86_64)
-- | --
Revision | 215b794a7c1f3e28151d0f7712e32c71be48775c
OS | macOS Version 14.7.2 (Build 23H311)
```

<img width="1305" alt="Image" src="https://github.com/user-attachments/assets/e9907852-4f65-4b35-9993-9609d963a772" />


Have tried:
- various resolutions
- various window sizes
- full screen vs not full screen


@JaneX8 could you try something when you get a chance?
1. Open brave://downloads
2. Right click somewhere on the page and select "Inspect" from the context menu:

<img width="214" alt="Image" src="https://github.com/user-attachments/assets/dbfd55a3-d2eb-4d8a-9e5c-543b45b4bc95" />

3. When the devtools open, near the top click on "Console"

<img width="1391" alt="Image" src="https://github.com/user-attachments/assets/bd4efe61-b1a4-4646-af8f-8171256258fe" />

4. Can you let us know if there are any errors displayed here? (You see mine doesn't show any in the above screenshot)

**Comment by JaneX8** [comment]
Hi,
I have very little time but I quickly checked what you asked. No errors are shown in the console. It's identical to your last screenshot. Maybe this helps, the button seems only clickable when you're effectively on top of the scroll bar in a tiny area:

https://github.com/user-attachments/assets/a3fe643e-feef-44d8-a228-20dedb72d3fe

**Comment by LaurenWags** [comment]
@JaneX8 this looks promising. I don't have that "managed" message so that could be why I can't reproduce on my set up. I'll give it a try and report back. Thank you!

**Comment by LegalizeNukes** [comment]
I would like to add that I also have the "managed" message. The problem may indeed be related to it.

**Comment by LaurenWags** [comment]
Thanks @JaneX8 @LegalizeNukes - I can reliably reproduce the issue when "Your browser is managed by your organization" message is shown on brave://downloads 👍🏻 

-----------

@emerick the steps I used to reproduce are:
1. Using `1.73.104` (current release), downloaded a bunch of files
2. Confirmed "Clear" works fine (used the "undo" action to get my list back)

https://github.com/user-attachments/assets/a20d4d2f-86b3-4942-b630-cad899ec6fd4

3. Closed the browser
4. Added a policy via terminal (ex. ran `defaults write com.brave.Browser BraveVPNDisabled -bool true` on macOS) to get the "managed" message
5. Relaunched the browser
6. On brave://downloads confirmed I can see the "Your browser is managed by your organization" message
7. Confirmed the "Clear" button doesn't work unless I go to the far right edge as mentioned by @JaneX8 - can see it doesn't highlight until getting to that far right edge. When I click there it does work and it can be "undone"

https://github.com/user-attachments/assets/144cdf5c-51ee-4773-ad69-1b4e52f2c9cd


To clean up/remove the policy I then closed the browser, removed the policy I had set via the terminal (ran `defaults delete com.brave.Browser BraveVPNDisabled`), relaunched and saw that the "managed" message was removed.

**Comment by MadhaviSeelam** [comment]
Indeed, reproduced the issue with a policy enabled i.e. "Your browser is managed by your organization" message is shown.  If a policy is not configured, working as expected in below build in Win 11 x64
```
Brave | 1.73.104 Chromium: 131.0.6778.204 (Official Build) (64-bit)
-- | --
Revision | 215b794a7c1f3e28151d0f7712e32c71be48775c
OS | Windows 11 Version 24H2 (Build 26100.2314)
```

**Comment by emerick** [comment]
I think we should probably hide the managed footnote on the downloads, as it's incompatible with how our toolbar is implemented. 

@fallaciousreasoning I think I could use some advice on how to hide `managed-footnote` just on brave://downloads. Not sure if I'm missing an obvious approach, but nothing I tried seems to work. Was hoping I could use a style injection.

**Comment by JaneX8** [comment]
My bet is something like 

```css
managed-footnote {
display: none !important;
}
```

I didn't test it.

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.75.159 Chromium: 132.0.6834.83 (Official Build) beta (64-bit)
-- | --
Revision | a7b6f5093a2dad31fc3f56241a7d55be255e69e2
OS | Windows 11 Version 24H2 (Build 26100.2605)
```


Installed 1.75.159
launched Brave
downloaded few files
opened Local Group Policy Editor via gpedit.msc (Win + R)
opened Administrative Templates >> Classic Administrative Templates (ADM)>> Brave >> Brave Software Settings
double clicked to `Disable Tor Connectivity` 
selected `Enabled` >> `Apply`
Confirmed `brave://settings/appearance` is shown the message "Your `browser is managed` by your organization"
returned `brave://downloads`
clicked `Clear all`

**Comment by btlechowski** [comment]
Verified with

Brave | 1.75.170 Chromium: 132.0.6834.111 (Official Build) beta (64-bit)
-- | --
Revision | ea6b6fa547193f9769ffb0068880a4f41e364864
OS | Linux


<details>

1. Installed 1.75.x
2. launched Brave
3. downloaded few files
4. opened Local Group Policy Editor via gpedit.msc (Win + R)
5. opened Administrative Templates >> Classic Administrative Templates (ADM)>> Brave >> Brave Software Settings
6. double clicked to `Disable Tor Connectivity` 
7. selected `Enabled` >> `Apply`
8. confirmed brave://downloads` do not show the message "Your `browser is managed` by your organization"
9. confirmed `brave://settings/appearance` is shown the message "Your `browser is managed` by your organization"
10. returned `brave://downloads`
11. clicked `Clear all`
12. confirmed "Clear all" button on brave://downloads works as expected for all three scenarios, i.e. when a policy `Enabled`, `Disabled` and `Not configured` 



</details>
